src/Symfony/Bundle/FrameworkBundle/Tests/Functional/Bundle/TestBundle/AutowiringTypes/AnnotationReader.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\AutowiringTypes;
+
+use Doctrine\Common\Annotations\Reader;
+
+class AnnotationReader
```

Comment:
I had the same idea, just wasn't sure if having many arguments in the future is not too controversial :) and I was also wondering about the name of the class :)... So should I do it?